spec/Cabin/Cabin.vspec

```diff
@@ -98,61 +98,71 @@ Lights:
 Door:
   type: branch
   instances:
-    - Row[1,4]
+    - Row[1,2]
```

Comment:
Adapting number of rows to changed number of seats.

But then I started to think about hatchback and station wagons. At least in some countries they are sometimes referred to as 3/5-door vehicles. 

I think we would benefit from making it clear in a comment in VSS if the rear door for hatchbacks and station wagons shall be considered as a door (and represented here) or being considered as a trunk (`Vehicle.Body.Trunk`). If we want to consider them as a door we should preferably give some guidance on what literal to use for position. E.g. by either having a station wagon as "default vehicle" and include the position "center", or just giving a comment that rear doot shall preferably have the position "center"


```
Door:
  type: branch
  instances:
    - Row[1,3]
    - ["Left", "Center", "Right"]
  description: All doors in the vehicle.

# Default value based on instance declaration above with 2 rows and 2 doors in each row.
DoorCount:
  datatype: uint8
  type: attribute
  default: 5 # (i.e. 2 in first row, 2 in second row, and then a rear door as part of third row)
  description: Number of doors in vehicle

```

But that would give us a specification with quite a lot of holes, e.g. there will be a `Row1.Center` in generated JSON/CSV even if not needed. The alternative is of course that we add a comment that rear door (if any) is to be handled

Reply:
At the end it could be a combination of the original instances in door and a second branch Door.Hatchback with instances `[left, center, right]`

Reply:
I think it boils down to what we need for the rear door. Is the open/locked-info part of Vehicle.Body.Trunk sufficient, or do we need all window/shade-information that you typically have for a door as well.